canonical-web-and-design/maas-ui
# feat(ui): scroll to top when changing tabs

## Done

- Scroll the window to the top when changing machine details tabs.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to machine details for a machine with test results.
- Scroll down to the bottom of the network card and click "View results ›".
- You should get scrolled to the top of the page.

## Fixes

Fixes: #2369.
